lib/rumoji/emoji/symbols.rb

```diff
@@ -198,10 +198,41 @@ class Emoji
       self.new("\u{0030 fe0f 20e3}" , [:zero]),
       self.new("\u{1f3fa}", [:black_circle_record]),
       self.new("\u{1f3f9}", [:black_square_stop]),
-      self.new("\u{1f3f8}", [:double_vertical_bar]),
```

Comment:
FYI: This Unicode code-point was changed because it was incorrectly representing "BADMINTON RACQUET AND SHUTTLECOCK" instead of "DOUBLE VERTICAL BAR".